test/videojs-hls_test.js

```diff
@@ -87,24 +87,24 @@ var
     player.buffered = function() {
       return videojs.createTimeRange(0, 0);
     };
-    mockTech(player.tech);
+    mockTech(player.tech_);
 
     return player;
   },
   openMediaSource = function(player) {
     // ensure the Flash tech is ready
-    player.tech.triggerReady();
+    player.tech_.triggerReady();
```

Comment:
Not for this PR but we should just focus on the tech for these tests in the future (i.e. don't create a player at all).